src/tinyformat.h

```diff
@@ -155,7 +155,7 @@ namespace tfm = tinyformat;
 #endif
 
 #ifdef __APPLE__
-// Workaround OSX linker warning: xcode uses different default symbol
+// Workaround OSX linker warning: Xcode uses different default symbol
```

Comment:
"OS X"

Reply:
Technically, it is macOS nowadays. There is no OS X anymore :)